kube/resources/volume-db.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: airbyte-volume-db
```

Comment:
airbyte-volume-job-db? with how we are structuring things we are likely to have more dbs in the future. the config db really is already one. quixotic naming is the only thing that has saved us there so far.

Reply:
I don't think it's likely even at scale that we'll multiple db servers for quite some time. We can easily have multiple dbs inside that server though.

Reply:
Ultimately users might want to use an external db anyway.

Reply:
i think my original comment may not have been clear based on the responses. still fine though if you are happy with how it is and want to leave it.

i think we should be naming our volumes based on _what they store_. ultimately it is then up to the developer to decide if they want to use the built in or an external db to store that data. the names `airbyte-volume-data`, `airbyte-volume-db`, and `airbyte-volume-workspace` are inconsistent right now. 2 effectively just mean volumes with some undefined data on them and the last describes what it stores. i like the naming convention on the last one best. i'd suggest: `airbyte-volume-config`, `airbyte-volume-job`, `airbyte-volume-workspace`. these now all describe what they store.

Reply:
I agree with `config` and `workspace`. However, the DB will not necessarily only store job data. From a customer ease of use perspective, I can't imagine a situation where they want to be running multiple DB servers (or if this is configurable with external DBs that they'd want to point to more than one db).

Reply:
Also at that point `job` vs `workspace` is a confusing distinction.